### PR TITLE
edi_af_aisf_ash_kom AFC-1310: new module that updates all employees c…

### PR DESCRIPTION
**Pull request by SebastianGieser-Vertel**
…onnected to all departments

### CODE REVIEW

File: edi_af_aisf_ash_kom/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    },
+    'data': [
+        'data/edi_route_data.xml',
+        'data/server_action.xml',
+        'views/edi_af_aisf_ash_kom_views.xml',
+    ],
+    'application': False,
+    'installable': True,
+}
+# vim:expandtab:smartindent:tabstop=4s:softtabstop=4:shiftwidth=4:
```
Comment (arturchilangwa-vertel) [inline edi_af_aisf_ash_kom/__manifest__.py]: Do we need config for Vim?

File: edi_af_aisf_ash_kom/data/edi_route_data.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<openerp>
```
Comment (arturchilangwa-vertel) [inline edi_af_aisf_ash_kom/data/edi_route_data.xml]: Is this old odoo tags, that can only be used in older versions? Should it not be `<odoo/>`.

File: edi_af_aisf_ash_kom/models/edi_route.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from odoo import models, fields, api, _
+
+import logging
+_logger = logging.getLogger(__name__)
```
Comment (arturchilangwa-vertel) [inline edi_af_aisf_ash_kom/models/edi_route.py]: Unused logger that can be removed

File: edi_af_aisf_ash_kom/messages/af_aisf_ash_kom_get_all.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+#
+##############################################################################
+from odoo import models, api, _
+import ast
+import json
+
+import logging
+
+_logger = logging.getLogger(__name__)
+
```
Comment (arturchilangwa-vertel) [inline edi_af_aisf_ash_kom/messages/af_aisf_ash_kom_get_all.py:27]: Unused logger that can be removed

**Review by arturchilangwa-vertel — CHANGES_REQUESTED** [review]

File: edi_af_aisf_ash_kom/messages/af_aisf_ash_kom_get_all.py (hunk 4 above)
Comment (SebastianGieser-Vertel) [reply edi_af_aisf_ash_kom/messages/af_aisf_ash_kom_get_all.py:27]: made some changes, now it's used

**Review by arturchilangwa-vertel — APPROVED** [review]

**Pull request merged**